lib/Constants.js

```diff
@@ -32,92 +32,125 @@ module.exports = {
       16: 'weather',
       17: 'irrigation',
       18: 'gas',
+      256: 'previous_event_cleared',
+      257: 'alert_location',
       512: 'type',
       513: 'level'
     }
 
     return alarmMap[index] || 'unknown_' + index
   },
   sensorType (index) { // https://github.com/OpenZWave/open-zwave/blob/0d94c9427bbd19e47457578bccc60b16c6679b49/config/SensorMultiLevelCCTypes.xml
-    switch (index) {
-      case 1:
-      case 11:
-      case 23:
-      case 24:
-      case 34:
-      case 62:
-      case 63:
-      case 64:
-      case 65:
-      case 72:
-      case 73:
-      case 74:
-      case 75:
-      case 76:
-      case 77:
-      case 80:
-        return 'sensor_temperature'
-      case 3:
-        return 'sensor_illuminance'
-      case 4:
-      case 15:
-      case 16:
-      case 28:
-      case 29:
-        return 'sensor_electricity'
-      case 5:
-      case 41:
-        return 'sensor_humidity'
-      case 6:
-      case 52:
-      case 53:
-      case 54:
-      case 8: // pressure
-      case 9: // pressure
-        return 'sensor_speed'
-      case 7:
-      case 21: // angle position
-        return 'sensor_direction'
-      case 10:
-      case 27: // uv index
-        return 'sensor_sun'
-      case 12: // rain rate
-      case 13: // tide level
-      case 19: // capacity
-      case 56:
-      case 57:
-        return 'sensor_water'
-      case 14:
-      case 46:
-      case 47:
-      case 48:
-      case 49:
-      case 51:
-      case 66:
-      case 67:
-      case 68:
-        return 'sensor_weight'
-      case 17:
-      case 40: // carbon monoxide
-      case 55:
-        return 'sensor_co2'
-      case 18:
-        return 'sensor_air'
-      case 22:
-        return 'sensor_frequency'
-      case 30:
-        return 'sensor_sound'
-      case 33:
-        return 'sensor_time'
-      case 44:
-      case 45:
-      case 50:
-      case 60:
-      case 69:
-        return 'sensor_heart'
-      default:
-        return null
+    let sensorMap = {
```

Comment:
There are some more sensors in the list, also I suggest to add another map with sensors icons so we can keep only one sensor generic configuration and edit the object_id and the discovery payload mdi icon